episodes/03-create.md

```diff
@@ -726,11 +726,15 @@ $ cp minotaur.dat unicorn.dat basilisk.dat
 
 ## Solution
 
-When `cp` is given two arguments and the second is a destination directory
-`cp` copies the files to the destination directory.
```

Comment:
It is helpful to distinguish between two and three arguments because for two arguments, if the second argument is not a directory but a filename, then the first file is copied and renamed.

Reply:
Having it in the main text is a good idea.

Reply:
Perhaps move this out of the challenge block and make it something that is demonstrated.